apps/frontend/src/app/app.component.scss

```diff
@@ -42,4 +42,27 @@ thead {
 
 td {
   padding: 10px;
+}
+
+button {
+  border:none;
+  border-radius: 50%;
+  padding: 10px;
+  background: no-repeat left center;
+  background-size: cover;
+}
+.btn-basic {
+  background-image:url('../assets/minus-svgrepo-com.svg') ;
```

Comment:
@JadSafah Alex did that on purpose! btn-pasic is that minus sign lol. it is KINDA odd, but everything's working ok! do we have other images perhaps? 🤔

Reply:
maybe we could do this one?
https://icons8.com/icons/set/sort

Reply:
oh ok! It looked too clean to be a mistake lol

Reply:
i LOVE this one emily! I wanted your input for the icons anyway :D :D :D :D